addons/mail/static/src/core/common/composer.scss

```diff
@@ -70,12 +70,15 @@
         &.o-sendMessageActive {
             background-color: rgba($o-action, .75);
             opacity: 100% !important;
+            &:focus-visible {
+                outline: $primary solid 1px;
```

Comment:
`$primary` (purple) and `$o-action` (green) doesn't go well together. Prefer `darken($o-action, 15%)` instead

Reply:
I made the change